packaging/Dockerfile

```diff
@@ -1,8 +1,6 @@
-FROM registry.opensource.zalan.do/pathfinder/archlinux:latest-SNAPSHOT
+FROM alpine
 MAINTAINER Skipper Maintainers <team-pathfinder@zalando.de>
-RUN pacman --noconfirm -Syyu
-RUN pacman --noconfirm -S git go
-RUN mkdir -p /opt/go-workspace
-ENV GOPATH /opt/go-workspace
-ENV PATH $PATH:$GOPATH/bin
-RUN go get github.com/zalando/skipper/...
+RUN mkdir -p /usr/bin
+ADD skipper /usr/bin/skipper
```

Comment:
I think this could be one line `ADD skipper eskip /usr/bin`. This would make the resulting container have one layer less